kinetic/distribution.yaml

```diff
@@ -2464,6 +2464,23 @@ repositories:
       url: https://github.com/tork-a/dynpick_driver.git
       version: master
     status: maintained
+  e100_base-relase:
+    doc:
+      type: git
+      url: https://github.com/gaitech-robotics/e100_base-release.git
+      version: kinetic-devel
```

Comment:
This failed CI.  The likely reason is that the doc and the source entry should point to the upstream sources, not the release repository.  If there is no upstream source repository, then just elide the doc and source entries.